content/guides/models/app/settings-page/billing-settings.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Manage your organization's billing settings
+menu:
+  default:
+    parent: settings
+title: Manage billing settings
+weight: 20
+---
+
+Navigate to your user profile page and select your user icon on the top right corner. From the dropdown, choose **Settings**, then click **Billing**.
+
+## Plan details
+
+### Teams plan
+The **Teams plan** section summarizes your organization's current plan, including a summary of monthly charges, as well as your current monthly usage:
+
+- Monthly charges.
+- Seats used. For details and a list of users, click **Manage users**.
+- Tracked hours used. For details, click **View usage**.
+- Amount of monthly data ingested by Weave. For details, click **View usage**.
+
+From here, you can compare plans or talk to Sales.
```

Comment:
Is there an email or something we can link to here?

Reply:
I don't think that would be useful here -- this is discussing a UI from which you can talk to Sales. I think we want them to use the UI!